colpali_engine/models/qwen2_5/colqwen2_5/modeling_colqwen2_5.py

```diff
@@ -14,32 +13,22 @@ class ColQwen2_5(Qwen2_5_VLForConditionalGeneration):  # noqa: N801
 
     main_input_name: ClassVar[str] = "doc_input_ids"  # transformers-related
 
-    def __init__(self, config: Qwen2_5_VLConfig):
+    def __init__(self, config: Qwen2_5_VLConfig, remove_context_embeddings: Optional[bool] = False):
+        """
+        Initializes the ColQwen2.5 model.
+
+        Args:
+        - config (Qwen2.5VLConfig): The model configuration.
```

Comment:
random thought but is this the correct docstring format consisytent with the rest ?